### ISSUE TITLE
Need to restrict top level data types to those specified in the standard

### ISSUE THREAD

**Issue by David-Chadwick** [w3c/vc-data-model#149]
The current data model does not place any restrictions on the top level data types/structures. The type parameter can have any value. Consequently anyone can define their own type (e.g. unverifiableDocument). This loophole should be closed. It is proposed to close it by introducing two standard data types: Verifiable Credential and Verifiable Profile (or just one, depending upon whether profiles are discontinued or not). This would change the current JSON structure to 
```
"verifiableCredential": {
     "id": "< credential id >",
     "type": ["ProofOfAgeCredential"]
```

**Comment by David-Chadwick** [comment]
The following solution is proposed that is aligned with JSON-LD.

Add a new section 5.1 to to 5 Basic Concepts

5.1. Type and ID

A verifiable credential MUST have a "type" property. The first value MUST be set to "VerifiableCredential". It MAY have additional values, and if present these MUST be set to the subtype of the verifiable credential.

A verifiable credential MUST have an "id" property. This property MUST be single valued and must be globally unique.

Similar text needs to be added to Section 5.5 Profile.

A verifiable profile MUST have a "type" property. The first value MUST be set to "VerifiableProfile". It MAY have additional values, and if present these MUST be set to the subtype of the VerifiableProfile.

**Comment by msporny** [comment]
I made an attempt to resolve this issue in https://github.com/w3c/vc-data-model/pull/189.

**Comment by msporny** [comment]
I've implemented these changes in PR #189 and PR #191. Do those PRs resolve this issue, @David-Chadwick?
